dist/range.js

```diff
@@ -62,6 +74,7 @@
         });
       }
     };
+    window.AlpineMagicHelpers = AlpineMagicHelpers;
```

Comment:
Won't this get clobbered if there's already `window.AlpineMagicHelpers`?

Eg. If you load 2 helpers

Reply:
Yeah true. I guess it would be the same though. I'll figure something else out. Or do you have another idea?

Reply:
Could merge with the existing object if it exists?